api.cpp

```diff
@@ -84,6 +84,16 @@ class Context {
     });
   }
 
+  val summary() {
+    return gpp_rethrow([=]() {
+      val result = val::object();
```

Comment:
Nit: I'd probably avoid the object and just return the string, converted to val, directly.